# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gmp/hotfix-multilib.patch
# Copyright (C) 2023 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Fix shared multi-lib headers for powerpc64/32 and sparc64/32 et al.

--- gmp-6.2.1/gmp-h.in.vanilla	2023-12-17 17:33:35.362754654 +0100
+++ gmp-6.2.1/gmp-h.in	2023-12-17 17:35:37.846749426 +0100
@@ -40,7 +40,7 @@
 #if ! defined (__GMP_WITHIN_CONFIGURE)
 #define __GMP_HAVE_HOST_CPU_FAMILY_power   @HAVE_HOST_CPU_FAMILY_power@
 #define __GMP_HAVE_HOST_CPU_FAMILY_powerpc @HAVE_HOST_CPU_FAMILY_powerpc@
-#define GMP_LIMB_BITS                      @GMP_LIMB_BITS@
+#define GMP_LIMB_BITS                      (__SIZEOF_POINTER__ * 8)
 #define GMP_NAIL_BITS                      @GMP_NAIL_BITS@
 #endif
 #define GMP_NUMB_BITS     (GMP_LIMB_BITS - GMP_NAIL_BITS)
